doc/_themes/sphinx13/static/sphinx13.css

```diff
@@ -338,7 +338,7 @@ div.admonition, div.warning {
     font-size: 0.9em;
     margin: 1em 0 1em 0;
     border: 1px solid #86989B;
-    border-radius: 2px;
+    border-radius: 3px;
```

Comment:
You should add in

```
div.admonition > p.admonition-title,
div.warning > p.admonition-title {
    ...
    border-radius: 3px, 3x, 0, 0;
}
```
oherwise, the title background overlays your border
![image](https://github.com/sphinx-doc/sphinx/assets/2836374/85a75503-ddf9-44da-9f33-1d109f528047)

Reply:
done cheers